.github/CODEOWNERS

```diff
@@ -11,6 +11,8 @@
 /plugins/cost-insights                      @backstage/silver-lining
 /plugins/cloudbuild                         @trivago/ebarrios
 /plugins/search                             @backstage/techdocs-core
+/plugins/search-*                           @backstage/techdocs-core
+/packages/app/src/components/search/*       @backstage/techdocs-core
```

Comment:
Let's not have the code in the app or this :p

Reply:
@iamEAP You're looking at making this a heavily composable / customisable thing, is that it? Sort of like the catalog?

Reply:
Yep!  We can focus that convo here: https://github.com/backstage/backstage/pull/4515#discussion_r582660072